app/pages/onboarding/04-connect-ledger/connect-ledger.tsx

```diff
@@ -70,7 +71,7 @@ export const ConnectLedger: React.FC = () => {
         setHasConfirmedAddress(true);
         await delay(750);
 
-        if (CONFIG.STX_NETWORK === 'mainnet' && !confirmedResponse.address.startsWith('SP')) {
+        if (isMainnet() && !confirmedResponse.address.startsWith('SP')) {
```

Comment:
does this ever need to be -> `!confirmedResponse?.address?.startsWith?.('SP')`?

Reply:
Above is code: 

```ts
if (confirmedResponse.address) {
```